gcn/notices/core/Pointing.schema.json

```diff
@@ -6,46 +6,43 @@
   "$comment": "Providers should $ref this schema directly but Attitudes.schema.json instead.",
   "type": "object",
   "properties": {
-    "instrument_pointing": {
-      "type": "object",
-      "properties": {
-        "ra": {
-          "type": "number",
-          "description": "RA pointing of the instrument Z axis"
-        },
-        "dec": {
-          "type": "number",
-          "description": "Dec pointing of the instrument Z axis"
-        },
-        "roll": {
-          "type": "number",
-          "description": "Roll of the instrument about its Z axis"
-        },
-        "position_angle": {
-          "type": "number",
-          "description": "Rotation angle [deg] between the spacecraft X-Z plane and the plane defined by the spacecraft X-axis and celestial North"
-        },
-        "q1": { "type": "number", "description": "Attitude quarternion" },
-        "q2": { "type": "number", "description": "Attitude quarternion" },
-        "q3": { "type": "number", "description": "Attitude quarternion" },
-        "q4": { "type": "number", "description": "Attitude quarternion" },
-        "rotation_x": {
-          "type": "number",
-          "description": "Rotational rate [rad/s] about the X axis"
-        },
-        "rotation_y": {
-          "type": "number",
-          "description": "Rotational rate [rad/s] about the Y axis"
-        },
-        "rotation_z": {
-          "type": "number",
-          "description": "Rotational rate [rad/s] about the Z axis"
-        },
-        "time": {
-          "type": "string",
-          "description": "Time [UTC, ISO 8601] of this data, if different than the trigger time."
-        }
-      }
+    "ra": {
+      "type": "number",
+      "description": "RA pointing of the instrument Z axis"
+    },
+    "dec": {
+      "type": "number",
+      "description": "Dec pointing of the instrument Z axis"
+    },
+    "roll": {
+      "type": "number",
+      "description": "Roll of the instrument about its Z axis"
+    },
+    "position_angle": {
+      "type": "number",
+      "description": "Rotation angle [deg] between the spacecraft X-Z plane and the plane defined by the spacecraft X-axis and celestial North"
+    },
+    "attitude_quarternion": {
+      "type": "array",
+      "items": {
+        "type": "number"
+      },
+      "minItems": 4,
+      "maxItems": 4,
+      "description": ""
```

Comment:
Representing attitude state of a spacecraft, using four element vector: a scalar and 3 element unit vector.